File: food-review/src/main/java/com/food/review/gateway/mongo/repository/ReviewRepository.java
package com.food.review.gateway.mongo.repository;

import com.food.review.gateway.mongo.entity.ReviewMongo;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

import java.util.UUID;

@Repository
public interface ReviewRepository extends MongoRepository<ReviewMongo, UUID> {
}

File: food-review/src/main/kotlin/com/food/review/gateway/mongo/repository/ReviewRepository.kt
package com.food.review.gateway.mongo.repository

import com.food.review.gateway.mongo.entity.ReviewMongo
import java.util.Optional
import java.util.UUID
import org.springframework.data.mongodb.repository.MongoRepository

interface ReviewRepository: MongoRepository<ReviewMongo, String> {

    fun findByUserUuidAndUuid(userUuid: String, uuid: String): Optional<ReviewMongo>

}